sceptre/synapseprod/templates/accounts.yaml

```diff
@@ -10,6 +10,31 @@ Resources:
   # 1. Detach or remove the following user resources: login profile, attached
   #    MFA device, access-keys, ssh-keys, and policies.
   # 2. Detach the user from all groups.
+  AWSIAMRedashSvcUser:
+    Type: AWS::IAM::User
```

Comment:
how about creating service users with org-formation going forward? do it in here  https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/master/org-formation/600-access/_tasks.yaml

Reply:
Sure can.